sessionctx/variable/tidb_vars.go

```diff
@@ -1083,6 +1085,7 @@ const (
 	DefTiDBOptRangeMaxSize                          = 0
 	DefTiDBCostModelVer                             = 1
 	DefTiDBServerMemoryLimitSessMinSize             = 128 << 20
+	DefTiDBEnableGOGCTuner                          = true
```

Comment:
Decided enable it default?

Reply:
Yes, it is to facilitate follow-up testing. we will disable it before the next release.